src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoMasterSchedulerStatusDao.java

```diff
@@ -0,0 +1,35 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import javax.annotation.Resource;
+
+import org.joda.time.DateTime;
+import org.sagebionetworks.bridge.dao.MasterSchedulerStatusDao;
+import org.sagebionetworks.bridge.models.DateTimeHolder;
+import org.springframework.stereotype.Component;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+
+@Component
+public class DynamoMasterSchedulerStatusDao implements MasterSchedulerStatusDao {
+    
+    static final String SCHEDULER_STATUS_HASH_KEY = "BridgeMasterScheduler";
+    
+    private DynamoDBMapper mapper;
+    
+    @Resource(name = "masterSchedulerStatusMapper")
+    final void setMapper(DynamoDBMapper mapper) {
+        this.mapper = mapper;
+    }
+
+    @Override
+    public DateTimeHolder getLastProcessedTime() {
+        DynamoMasterSchedulerStatus statusKey = new DynamoMasterSchedulerStatus();
+        statusKey.setHashKey(SCHEDULER_STATUS_HASH_KEY);
+        
+        DynamoMasterSchedulerStatus status = mapper.load(statusKey);
+        if (status == null) {
+            return null;
+        }
+        return new DateTimeHolder(new DateTime(status.getLastProcessedTime()));
+    }
+}
```

Comment:
This will work. There may be an annotation that you can put on DynamoMasterSchedulerStatus which will allow you to have the java property be a DateTime value, but the persisted in Dynamo value to be a number (a long). This is one annotation and would allow you to return status.getLastProcessedTime or even just status from the Dao, in case we augment status in the future and need to use other properties from it. The Holder object could be introduced in the controller or service just to wrap the date to make a nice JSON payload.

Or, you can just do this... this works.